tests/gen/gen_auto.go

```diff
@@ -0,0 +1,246 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package gen
+
+import (
+	"math/rand"
+
+	"github.com/sourcenetwork/defradb/client"
+	"github.com/sourcenetwork/defradb/client/request"
+)
+
+const (
+	defaultNumDocs           = 10
```

Comment:
suggestion: It might be nicer if these were public, particularly if we are considering moving this tool out of the test directory.

todo: Public or private, I think these consts could benefit from documentation.